### ISSUE TITLE
Docker/swarm API authentication - user/passwd credentials

### ISSUE THREAD

**Issue by bluishpe** [docker/swarm#1667]
Hello,
    Do you know if its possible to access docker or swarm only after entering username and password for it? Is there any way to do it?

Regards

**Comment by nishanttotla** [comment]
@bluishpe this looks like a duplicate of #1511, so I'm closing it. If it's different, please let us know.
